services/web/server/tests/sandbox/viewers_openapi_generator.py

```diff
@@ -0,0 +1,66 @@
+# pylint: skip-file
+
+from typing import List, Optional
+
+import yaml
+from fastapi import FastAPI
+from fastapi.routing import APIRoute
+from pydantic import BaseModel, Field
+from pydantic.networks import HttpUrl
+from pydantic.types import PositiveInt
+from yaml import safe_dump
+
+app = FastAPI()
+
+
+
+# TODO:
+#class ErrorType(BaseModel):
+#    logs: List[LogMessageType] = []
+#    errors: List[ErrorItemType]
+#    status: int
```

Comment:
Will this be used? Please add some reference for the reason it is here or remove it.

Reply:
I will cleanup but this is an un-official geneartor therefore it is kept in the sandbox ...